packages/tests/src.ts/test-providers.ts

```diff
@@ -1217,6 +1217,11 @@ describe("Test WebSocketProvider", function() {
         const provider = ethers.providers.InfuraProvider.getWebSocketProvider();
         await testWebSocketProvider(provider);
     });
+
+    it('Can create Provider with connection-info', async function() {
+        const provider = new ethers.providers.WebSocketProvider({ url: 'http://test.com' });
```

Comment:
For some reason this still flags as invalid in my editor, allthough types seem updated when I click towards it 😅